pandas/tests/io/test_feather.py

```diff
@@ -1,17 +1,17 @@
 """ test feather-format compat """
-
```

Comment:
was this not being linted before?

Reply:
Just looked at config.cfg.  The flake complaint being fixed here is E402 (module-level import not at top of module), which is one of two specifically ignored in setup.cfg.  Woops. Just moved the ignore spec from lint.sh to cfg, will stop messing with this until further notice.